### ISSUE TITLE
Error while ES6 importing 'prettier-plugin-rust' in JavaScript file.

### ISSUE THREAD

**Issue by biraj21** [jinxdash/prettier-plugin-rust#2]
# Bug Report

I am using prettier and its plugins (`prettier-plugin-java, @prettier/plugin-php`) in Node.js to format each file based on its inferred parser. Java & PHP are working fine with ES6 imports but I am getting the error while importing `prettier-plugin-rust`. It is working fine in Common JS modules though.

### 💻 Input

```js
// index.js
import prettier from "prettier";
import javaPlugin from "prettier-plugin-java";
import phpPligin from "@prettier/plugin-php";
import rustPugin from "prettier-plugin-rust";
```

### 🔴 Output

```
node:internal/process/esm_loader:94
    internalBinding('errors').triggerUncaughtException(
                              ^

Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/mnt/Data/code/formatter_test/node_modules/prettier/doc' imported from /mnt/Data/code/formatter_test/node_modules/prettier-plugin-rust/index.js
Did you mean to import prettier/doc.js?

    at new NodeError (node:internal/errors:372:5)
    at finalizeResolution (node:internal/modules/esm/resolve:437:11)
    at moduleResolve (node:internal/modules/esm/resolve:1009:10)
    at defaultResolve (node:internal/modules/esm/resolve:1218:11)
    at ESMLoader.resolve (node:internal/modules/esm/loader:580:30)
    at ESMLoader.getModuleJob (node:internal/modules/esm/loader:294:18)
    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:80:40)
    at link (node:internal/modules/esm/module_job:78:36) {
  code: 'ERR_MODULE_NOT_FOUND'
}
```

Common JS imports are working fine though.

```js
const prettier = require("prettier");
const javaPlugin = require("prettier-plugin-java");
const phpPligin = require("@prettier/plugin-php");
const rustPlugin = require("prettier-plugin-rust");
```

**Comment by jinxdash** [comment]
I just published 0.1.5, should be good~

**Comment by biraj21** [comment]
I updated the package and it is working fine now.

```js
import prettier from "prettier";
import javaPlugin from "prettier-plugin-java";
import phpPligin from "@prettier/plugin-php";
import * as rustPlugin from "prettier-plugin-rust";

const plugins = [javaPlugin, phpPligin, rustPlugin];
```

Thank you!
